source/extensions/filters/http/admission_control/admission_control.cc

```diff
@@ -52,6 +52,11 @@ double AdmissionControlFilterConfig::aggression() const {
 
 double AdmissionControlFilterConfig::successRateThreshold() const {
   const double pct = sr_threshold_ ? sr_threshold_->value() : defaultSuccessRateThreshold;
+  // The threshold value should not be 0. Since you cannot add a validation on a widely used Runtime
+  // percent value, simply map a 0 percent for the field to the default success rate threshold.
+  if (pct == 0) {
+    return defaultSuccessRateThreshold;
```

Comment:
`defaultSuccessRateThreshold` is expressed as a percentage, and therefore this line seems to return `95.0` and not `0.95`. This is different from the behavior resulted in by the first occurrence of `defaultSuccessRateThreshold` in this member function definition.